pkg/detectors/slackwebhook/slackwebhook.go

```diff
@@ -62,8 +65,18 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 					continue
 				}
 				body := string(bodyBytes)
-				if (res.StatusCode >= 200 && res.StatusCode < 300) || (res.StatusCode == 400 && (strings.Contains(body, "no_text") || strings.Contains(body, "missing_text"))) {
-					s1.Verified = true
+
+				if err == nil {
+					defer res.Body.Close()
+					if res.StatusCode >= 200 && res.StatusCode < 300 || (res.StatusCode == 400 && (strings.Contains(body, "no_text") || strings.Contains(body, "missing_text"))) {
```

Comment:
out of curiosity, do you know what these magic strings are for? do they factor into the question of whether the candidate secret is live?

Reply:
not sure myself - saw that they were part of the pre-existing verification logic and that it was written by a contractor so didn't have someone i could DM to ask